torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -3271,11 +3273,26 @@ def parse_env(var):
             self.assertEqual(ddp_logging_data.nccl_debug, parse_env("NCCL_DEBUG"))
             self.assertEqual(ddp_logging_data.nccl_nthreads, parse_env("NCCL_NTHREADS"))
             self.assertEqual(ddp_logging_data.nccl_ib_timeout, parse_env("NCCL_IB_TIMEOUT"))
+            # test runtime logging fields
+            self.assertEqual(ddp_logging_data.unused_parameter_size, 0)
+            self.assertEqual(ddp_logging_data.has_rebuilt_buckets, True)
+            self.assertEqual(ddp_logging_data.rebuilt_bucket_sizes, [param_size])
+            # It is hard to test accurate latency, but it can test whether the latency is
+            # a valid value and in the expected range.
+            self.assertGreaterEqual(ddp_logging_data.avg_forward_compute_time, 10)
+            self.assertGreaterEqual(ddp_logging_data.avg_backward_compute_comm_overlap_time, 10)
```

Comment:
Is this unit in ns? If so, how are we sure overalp time is >= 10ns, seems like this could be flaky?

Reply:
in ns, we can change this value to 1...